packages/expect-utils/src/utils.ts

```diff
@@ -390,12 +390,16 @@ export const arrayBufferEquality = (
   a: unknown,
   b: unknown,
 ): boolean | undefined => {
-  if (!(a instanceof ArrayBuffer) || !(b instanceof ArrayBuffer)) {
-    return undefined;
-  }
+  let dataViewA = a as DataView;
+  let dataViewB = b as DataView;
```

Comment:
Hm.. I’m not happy to see this cast. Right now it works with the following logic, but if the logic will change TS will fail to catch any errors. Would be useful to rethink this part.